package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "gulp": "^4.0.2",
     "js-yaml": "^4.1.0",
-    "markdown-link-check": "3.10.3",
+    "markdown-link-check": "^3.11.2",
```

Comment:
the version was pinned due to a bug in the tool, which is fixed now. See https://github.com/open-telemetry/opentelemetry-specification/pull/3327